modules/cnf-topology-aware-lifecycle-manager-installation-web-console.adoc

```diff
@@ -13,7 +13,7 @@ You can use the {product-title} web console to install the {cgu-operator-full}.
 // Based on polarion test cases
 
 * Install the latest version of the {rh-rhacm} Operator.
-* Set up a hub cluster with disconnected regitry.
```

Comment:
@Missxiaoguo - I removed the disconnected registry part. I thought it might be an error. Is it correct to remove this?

Reply:
Since this is for far edge and the concext is all around in a disconnected environment, this should be kept as-is.

Reply:
I've added it back in, thanks. I was a bit confused as there is a [disconnected](https://docs.openshift.com/container-platform/4.15/scalability_and_performance/ztp_far_edge/ztp-talm-updating-managed-policies.html) section in the docs too. :confused: